### PR TITLE
docs: sd-table-docs [skip chromatic]

### PR DESCRIPTION
**Pull request by smfonseca**
## Definition of Reviewable:

- [ ] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated
- [ ] Stories (features, a11y) are created/updated
- [ ] relevant tickets are linked

### CODE REVIEW

**Review by Vahid1919 — APPROVED** [review]

File: packages/components/src/styles/table/table.stories.ts [hunk 1]
```diff
@@ -1,29 +1,53 @@
...
 
 const { argTypes, parameters } = storybookDefaults('sd-table');
 const { overrideArgs } = storybookHelpers('sd-table');
 const { generateTemplate } = storybookTemplate('sd-table');
 
 /**
- * The `sd-table` component resets the styles of a table to predefined values. To style table cells use the `sd-table-cell` component.
+ * Used to reset the styles of a table to predefined values. To style table cells use the `sd-table-cell` component.
+ *
+ * **Related components**:
```
Comment (coraliefeil) [inline packages/components/src/styles/table/table.stories.ts:12]: Add:

Related templates
Table

Comment (smfonseca) [reply packages/components/src/styles/table/table.stories.ts]: Updated

**Comment by coraliefeil** [comment]
Pls left align text
We need horizontal top and bottom borders ... required by brand department

image

**Comment by coraliefeil** [comment]
Approved

**Comment by smfonseca** [comment]
> Pls left align text We need horizontal top and bottom borders ... required by brand department
> 
> image

Table headers aligned.
As discussed in teams, will not add the borders in the rows

**Comment by coraliefeil** [comment]
Approved

**Pull request merged**